lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -329,6 +329,8 @@ namespace Js
     //----------------------------------------------------------------------------
     Var JavascriptConversion::ToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * requestContext)
     {
+        Var result;
```

Comment:
Also should initialize to nullptr.

---
In reply to: [86454973](https://github.com/Microsoft/ChakraCore/pull/1911#discussion_r86454973) [](ancestors = 86454973)